content/en/apps/guides/hosting/vertical-vs-horizontal.md

```diff
@@ -0,0 +1,85 @@
+    apps/guides/hosting/4.x/data-migration
+    core/overview/architecture/
+---
+
+## Introduction
+
+CHT Core 4.0.0 introduces [a new architecture]({{< relref "core/overview/architecture" >}}) for hosting.  Specifically, 4.0.0 gives the ability to add more servers, known as horizontally scaling, so that large deployments can support more users.  This is a big improvement over naively adding more RAM and CPU to a single server where the CHT Core is running, known as vertically scaling.
+
+## Vertical scaling in 3.x and 4.x
+
+Before getting into how the CHT horizontally scales, it should be well understood the importance of vertical scaling and what it is.  This is the ability of the CHT to support more users by adding more RAM and CPU to either the bare-metal or virtual machine host. This ensures key services like API, Sentinel and, most importantly, CouchDB, can operate without performance degradation. 
+
+When thousands of users are simultaneous trying to synchronize with the CHT, the concurrency can sometimes overwhelm CouchDB. As discovered [through extensive research](https://forum.communityhealthtoolkit.org/t/how-we-tested-scalability-of-cht-infrastructure/1532) and [large production deployments](https://github.com/medic/cht-core/issues/8324#issuecomment-1691411542), administrators will start to see errors in their logs and end users will complain of slow sync times.  Before moving to more CouchDB nodes, administrators should consider adding more RAM and CPU to the single server where the CHT is hosted. This applies to both CHT 3.x and CHT 4.x. Given the easy of allocating more resources, presumably in virtualized environment like [EC2](https://aws.amazon.com/ec2/), [Proxmox](https://www.vmware.com/content/vmware/vmware-published-sites/us/products/esxi-and-esx.html.html) or [ESXi](https://www.vmware.com/content/vmware/vmware-published-sites/us/products/esxi-and-esx.html.html), this is much easier than moving to [from a single to multi-node CouchDB instance]({{< relref "apps/guides/hosting/4.x/data-migration" >}}).
+
+Here we see a normal deployment following our [bare minimum]({{< relref "apps/guides/hosting/requirements" >}}) hosting requirements for hosting the CHT.  We'll call this a "short" deployment because it is not yet vertically scaled:
+
+```mermaid
+flowchart TD
+
+subgraph couch1[" CouchDB - Single ''short'' Node "]
+    couchInner1["2 CPU/4 GB RAM"]
+end
+
+API["API"] --> HAProxy -->  couch1
+```
+
+After looking at the logs, and seeing error messages about API timeouts to CouchDB, the CHT admin can make this "taller" by adding both more RAM and CPU so it looks like this:
+
+```mermaid
+flowchart TD
+
+subgraph couch2[" CouchDB - Single ''tall'' Node "]
+    couchInner2["16 CPU/16 GB RAM"]
+end
+
+API["API"] --> HAProxy -->  couch2
+```
+
+Since both CHT 3.x and 4.x support this, vertical scaling is an easy, good first step in addressing performance issues in the CHT.
+
+## Horizontal scaling 4.x
+
```

Comment:
I think we need to add something about what to do if you're starting a new project as opposed to migrating. Right now the doc reads like you should start with single node, then vertically scale, then horizontally scale, but if you know your instance needs to support many thousands of users we'd recommend horizontally scaling from day 1 to avoid the migration later (eg: Uganda MoH).

Reply:
good idea! fixed in 647f818